scripts/Makefile.am

```diff
@@ -21,11 +21,17 @@ EXTRA_DIST = \
 	zfs2zol-patch.sed \
 	zol2zfs-patch.sed
 
+if BUILD_FREEBSD
+libtool = /usr/local/bin/libtool
+else
+libtool = /usr/bin/libtool
+endif
```

Comment:
Realistically, we probably don't need to specify the full path and could simply call `libtool` to minimize differences.